manifest/kgp/kgp-5.xml

```diff
@@ -0,0 +1,115 @@
+        </content:text>
+        <content:text i18n-id="{{uuid}}">Thank you for dying on the cross to pay for my sin.
+        </content:text>
+        <content:text i18n-id="{{uuid}}">I agree to turn from my old ways and follow you as my Savior and Lord. Please come and take first place in my life.
+        </content:text>
+        <content:text i18n-id="{{uuid}}">Could you say this to God and mean it?
+        </content:text>
+        <content:text i18n-id="{{uuid}}">Is there anything stopping you from saying this right now?
+        </content:text>
+      </paragraph>
+    </card>
+    <card>
+        <label>
+            <content:text i18n-id="{{uuid}}">Did you pray this prayer?</content:text>
+        </label>
+        <paragraph>
+            <content:text i18n-id="{{uuid}}">"Now this is eternal life: that they may know You, the only true God, and Jesus Christ, whom You have sent".
+            </content:text>
+            <content:text>John 17:3
+            </content:text>
+        </paragraph>
+        <paragraph>
+            <content:button type="event" value="kgp-6">
+                <content:text i18n-id="{{uuid}}">Not Ready</content:text>
+            </content:button>
+            <content:button type="event" value="followup-form">
+                <content:text i18n-id="{{uuid}}">Yes</content:text>
+            </content:button>
+            <content:link events="followup-form">
+                <content:text i18n-id="{{uuid}}">I already made this decision</content:text>
+            </content:link>
+        </paragraph>
+    </card>
+    <card hidden="true" listener="followup-form">
+        <label>
+            <content:text i18n-id="{{uuid}}">Jesus has come into your life as he promised.</content:text>
+        </label>
+        <paragraph>
+            <content:text i18n-id="{{uuid}}">Knowing someone better helps a relationship grow. Would you like to sign up for an email series that can help guide you in your new relationship with Jesus Christ?
+            </content:text>
+        </paragraph>
+        <form>
+            <content:input type="text" name="name">
+                <content:label>
+                    <content:text i18n-id="{{uuid}}">Name</content:text>
+                </content:label>
+                <content:placeholder>
+                    <content:text i18n-id="{{uuid}}">First Name and Last Name</content:text>
+                </content:placeholder>
+            </content:input>
+            <content:input type="text" name="email">
+                <content:label>
+                    <content:text i18n-id="{{uuid}}">Email</content:text>
+                </content:label>
+                <content:placeholder>
+                    <content:text i18n-id="{{uuid}}">Email</content:text>
+                </content:placeholder>
+            </content:input>
+            <content:button type="event" value="followup:send,send-information-modal">
```

Comment:
there is a namespace concept we incorporated in event id's, this is an external event from the package content and was the same event used in previous apps when submitting a followup form.